docs/maintain-sync.md

```diff
@@ -22,6 +22,16 @@ Archive nodes are used by utilities that need past information - like block expl
 
 Another type of node is a **light node**. A light node has only the runtime and the current state, but does not store past extrinsics and so cannot restore the full chain from genesis. Light nodes are useful for resource restricted devices. An interesting use-case of light nodes is a Chrome extension which is a node in its own right, running the runtime in WASM format: https://github.com/paritytech/substrate-light-ui
 
+Windows Fast Full Node Install Instructions:
+
+Install WSL: https://docs.microsoft.com/en-us/windows/wsl/install-win10
+Install Ubuntu (same webpage): https://docs.microsoft.com/en-us/windows/wsl/install-win10
+Download Polkadot binary within Ubunto by running:
+curl -sL https://github.com/paritytech/polkadot/releases/download/v0.7.28/polkadot -o polkadot
+run the following: sudo chmod +x polkdadot
+run the following: ./polkadot --name "Your Node Name Here"
+Find your node at https://telemetry.polkadot.io/#list/Kusama
```

Comment:
This isn't the end - it'll take a day or two for the node to sync with the network. Only then will the node become usable.